playbook/app/pb_kits/playbook/pb_circle_icon_button/docs/_circle_icon_button_link.html.erb

```diff
@@ -10,3 +10,11 @@
   link: "https://google.com",
   new_window: true
 }) %>
+<br/>
+<%= pb_rails("circle_icon_button", props: {
+  icon: "info",
+  variant: "secondary",
+  aria: { label: "Link to Playbook in new window" },
+  link: "https://playbook.powerapp.cloud/",
+  target: "child"
+}) %>
```

Comment:
Looks awesome @Lsimmons98 ! Can we duplicate this doc example for the React side? This last one with "target = child" is missing on the react side